paddlepaddle/paddlepaddle-native/build.cmd

```diff
@@ -7,7 +7,7 @@
 set FILEPATH="paddle"
 
 if "%1" == "cpu" (
-  set DOWNLOAD_URL="https://paddle-wheel.bj.bcebos.com/2.0.0/win-infer/open/cpu/paddle_inference_install_dir.zip"
+  set DOWNLOAD_URL="https://alpha-djl-demos.s3.amazonaws.com/temp/paddle_inference_install_dir-windows-openblas-2.0.2.zip"
```

Comment:
We should put into resource.djl.ai s3 buket

Reply:
This is just the download dir, used for native publish